commcare_connect/events/views.py

```diff
@@ -34,9 +35,24 @@ def create(self, request, *args, **kwargs):
 
         serializer = self.get_serializer(data=request.data, many=True)
         serializer.is_valid(raise_exception=True)
-
-        event_objects = [Event(**item) for item in serializer.validated_data]
-        Event.objects.bulk_create(event_objects)
+        try:
+            event_objects = [Event(**item) for item in serializer.validated_data]
```

Comment:
its generally safer to be explicit about incoming fields in case the phone adds unexpected ones, and to allow it to intentionally send fields you are not including (like the event id we have discussed).

Reply:
I have listed the fields here explicitly https://github.com/dimagi/commcare-connect/pull/349/files#diff-a7c344d1bdc227452ddf03733e8218375041272c8e3c6157bb96c11a7fa195ebR23 (May be I am not getting your point)

Reply:
This might have been my ignorance about the serializer. What happens if there are unexpected fields sent up? Are they ignored or raise an error?